pkg/networkservice/chains/nsmgr/server.go

```diff
@@ -78,7 +77,8 @@ type nsmgrServer struct {
 
 type serverOptions struct {
 	authorizeServer networkservice.NetworkServiceServer
-	connectOptions  []connect.Option
+	dialOptions     []grpc.DialOption
```

Comment:
This was done to allow us add new `connect` options in the future without modifying NSC, NSMgr chains constructors. So probably it would be better to keep `connectOptions` (or some other name) instead of specifying exact options?

Reply:
You can still add new options here... you just get more clarity... and not every connect option necessarily should be exposed up through NSMgr (though the ones so far do).